website/Makefile

```diff
@@ -1,54 +1,34 @@
-# Default: run this if working on the website locally to run in watch mode.
-website:
-	@echo "==> Downloading latest Docker image..."
-	@docker pull docker.mirror.hashicorp.services/hashicorp/vagrant-website
-	@echo "==> Starting website in Docker..."
-	@docker run \
-		--interactive \
-		--rm \
-		--tty \
-		--workdir "/website" \
-		--volume "$(shell pwd):/website" \
-		--volume "/website/node_modules" \
-		--publish "3000:3000" \
-		docker.mirror.hashicorp.services/hashicorp/vagrant-website \
-		npm start
+.DEFAULT_GOAL := website
 
-# This command will generate a static version of the website to the "out" folder.
-build:
-	@echo "==> Downloading latest Docker image..."
-	@docker pull docker.mirror.hashicorp.services/hashicorp/vagrant-website
-	@echo "==> Starting build in Docker..."
-	@docker run \
-		--interactive \
+PWD=$$(pwd)
+DOCKER_IMAGE="hashicorp/dev-portal"
+DOCKER_IMAGE_LOCAL="dev-portal-local"
+DOCKER_RUN_FLAGS=-it \
+		--publish "3000:3000" \
 		--rm \
 		--tty \
-		--workdir "/website" \
-		--volume "$(shell pwd):/website" \
-		--volume "/website/node_modules" \
-		docker.mirror.hashicorp.services/hashicorp/vagrant-website \
-		npm run static
+		--volume "$(PWD)/content:/app/content" \
+		--volume "$(PWD)/public:/app/public" \
+		--volume "$(PWD)/data:/app/data" \
+		--volume "$(PWD)/redirects.js:/app/redirects.js" \
+		--volume "next-dir:/app/website-preview/.next" \
+		--volume "$(PWD)/.env:/app/.env" \
```

Comment:
Does `website/.env` need any special env vars for the preview?

Reply:
It shouldn't! `DEV_IO` and `IS_CONTENT_PREVIEW` are set during build and should get picked up by dev-portal's next.config.js.